src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1981,6 +1981,7 @@ internal enum ErrorCode
         ERR_FileScopedAndNormalNamespace = 8955,
         ERR_FileScopedNamespaceNotBeforeAllMembers = 8956,
         ERR_NoImplicitConvTargetTypedConditional = 8957,
+        ERR_Newlines_are_not_allowed_inside_a_non_verbatim_interpolated_string = 8958,
```

Comment:
Please consider using the existing naming style - something like `ERR_NewlinesNotAllowedInNonVerbatimInterpolatedString`.

Reply:
sure.  though i personally prefer the form i used here (also used uniformly in the ide codebase) because you can tell the casing of stuff.  though that matters more for us as we have a lot of places where we need to use Title Case  for headers and whatnot.

Reply:
ok.  renamed.